### PR TITLE
Introduce example for testing the relevant words analysis

### LINKED ISSUE #1041: Build example for testing the prominent words analysis
We should build an example for testing the prominent words analysis. After inputting a locale and a text, a table with prominent words should be shown.

### PR DESCRIPTION
**Pull request by IreneStr**
## Summary
Introduce example for testing the relevant words analysis

## Test instructions

This PR can be tested by following these steps:

* Open the example in your browser
* Input a locale (currently only English or German)
* Input a text
* A prominent word table should be shown

Fixes #1041

### CODE REVIEW

File: examples/relevant-words-example/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <title>Yoast Prominent Words Analysis</title>
+    <link rel='stylesheet' id='source-sans-pro-css'
+          href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600&amp;subset=latin'/>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="../../dist/yoast-seo.min.css">
+
+    <script type="text/javascript" src="relevant-words-example-browserified.js"></script>
+</head>
+<body class="purple seo">
+
```
Comment (terw-dan) [inline examples/relevant-words-example/index.html]: I'd remove these whitespaces and indent the div, so it is clear it is a child of the body.

File: examples/relevant-words-example/relevant-words-example.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+let getRelevantCombinations = relevantWords.getRelevantCombinations;
+let WordCombination = require( "../../js/values/WordCombination" );
+let getWords = require( "../../js/stringProcessing/getWords" );
+var template = require( "../../js/templates.js" ).relevantWords;
+
+let map = require( "lodash/map" );
+let forEach = require( "lodash/forEach" );
+
+// Binds the renewData function on the change of inputelements.
+let bindEvents = function( ) {
```
Comment (terw-dan) [inline examples/relevant-words-example/relevant-words-example.js]: Please add documentation JSdoc style.

File: examples/relevant-words-example/relevant-words-example.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+let formatNumber = function ( number ) {
+
+	if ( Math.round( number ) === number ) {
+		return number;
+	}
+
+	return Math.round( number * 10000 ) / 10000;
+};
+
+let calculateRelevantWords = function () {
```
Comment (terw-dan) [inline examples/relevant-words-example/relevant-words-example.js]: Please add documentation to this function and the following.

**Comment by terw-dan** [comment]
CR 2 👍

**Comment by andizer** [comment]
Acceptance done 👍

**Pull request merged**